main.py

```diff
@@ -0,0 +1 @@
+print("Hello, World!")
```

Comment:
Please add option to pass argument (optional name) to your appication via command line. So that you can use your application to greet someone more specific.

sys.argv may help you. 

Please check that all the following scenarios work:

#default behavior: Hello, World!
python3 main.py

python3 main.py Alex

python3 main.py "dear friend"